tsconfig.json

```diff
@@ -6,7 +6,7 @@
     // "incremental": true,                   /* Enable incremental compilation */
     "target": "ES2017",                       /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019', 'ES2020', or 'ESNEXT'. */
     "module": "commonjs",                     /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', 'es2020', or 'ESNext'. */
-    "lib": ["ESNext"],                        /* Specify library files to be included in the compilation. */
+    "lib": ["ESNext", "WebWorker"],           /* Specify library files to be included in the compilation. */
```

Comment:
Do we need this here?

Reply:
Yes, the `self` in `main-browser` isn't available otherwise (the constant is only available in a webworker context).